src/main/java/com/offdk/play/model/game/MatchPlayer.java

```diff
@@ -7,28 +7,29 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
+import com.offdk.play.model.slack.User;
 import com.offdk.play.rating.RatingType;
 
 import io.vavr.control.Try;
 
 public class MatchPlayer {
-  private final String id;
+  private final User user;
   private final MatchPlayerType type;
   private Optional<Integer> score;
   private Optional<BigDecimal> rating;
 
   // Needed to allow spring-data to serialize data to object
   @PersistenceConstructor
-  MatchPlayer(String id, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
-    this.id = id;
+  MatchPlayer(User user, MatchPlayerType type, Optional<Integer> score, Optional<BigDecimal> rating) {
+    this.user = user;
     this.type = type;
     this.score = score;
     this.rating = rating;
   }
 
   @VisibleForTesting
```

Comment:
Aren't we using mongo? Is it really that difficult to store each rating as it's own data field within a map? Even if we don't add a second rating system, I could easily see a need to have an overall rating and a seasonal rating, at the same time.